bin/main.rs

```diff
@@ -132,6 +133,23 @@ struct Args {
         default_value = "120"
     )]
     cleanup_interval: u64,
+
+    /// The API topic prefix to use. This is an advanced setting that should only be used if you
+    /// know what you are doing
+    #[arg(
+        long = "api-prefix",
+        env = "WADM_API_PREFIX",
+        default_value = DEFAULT_WADM_TOPIC_PREFIX
+    )]
+    api_prefix: String,
```

Comment:
When would this and the below bucket option be overridden?

Reply:
We use our own prefix for things in Cosmonic all the time, so I figured it would be good to build it in. As for the bucket name, that could happen all the time for production as the guidance will be to preprovision your own buckets